using-mobile-extensions/adobe-journey-optimizer/README.md

```diff
@@ -80,164 +87,199 @@ Follow these steps to integrate the Adobe Journey Optimizer extension.
+    }];
+
+    ...
 }
 ```
+
 {% endtab %}
+
+{% tab title="iOS (ACP 2.x)" %}
+
+This extension is built on the AEPCore (3.x) and it is not compatible with ACPCore (2.x). Please follow [the guide for migrating to the Swift AEPCore](https://aep-sdks.gitbook.io/docs/resources/migrate-to-swift).
+
+{% endtab %}
+
 {% endtabs %}
 
 #### Sync the user push token with Adobe
 
-After importing and registering the extensions with your application, you need to sync the push token with Adobe Experience Platform, by using the setPushIdentifier API.
+After importing and registering the extensions with your application, you need to sync the push token with Adobe Experience Platform, by using the `setPushIdentifier` API.
 
 {% tabs %}
+
 {% tab title="Android" %}
-To retrieve the push token from Firebase Messaging Service, please read the tutorial on [retrieving the registration token](https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token) within the Firebase documentation. After retrieving the push token, you can use the following API to sync it with Profile in Platform.
 
-### Java
+To retrieve the push token from Firebase Messaging Service, read the tutorial on [retrieving the registration token](https://firebase.google.com/docs/cloud-messaging/android/client#retrieve-the-current-registration-token) within the Firebase documentation. After retrieving the push token, use the following API to sync it with the Profile in AEP.
 
-#### Syntax
+#### Java
+
+**Syntax**
 
 ```java
 public static void setPushIdentifier(final String pushIdentifier);
 ```
 
-| **Variable** | **Type** | **Description** |
+| Variable | Type | Description |
 | :----------- | :------- | :-------------- |
 | `pushIdentifier` | String | The push token value. |
 
-#### Example
+**Example**
 
 ```java
-FirebaseMessaging.getInstance().getToken()
-        .addOnCompleteListener(new OnCompleteListener<String>() {
-            @Override
-            public void onComplete(@NonNull Task<String> task) {
-                if (task.isSuccessful()) {
-                    String token = task.getResult();
-                    MobileCore.setPushIdentifier(token);
-                }
-            }
-        });
+FirebaseMessaging.getInstance().getToken().addOnCompleteListener(new OnCompleteListener<String>() {
+    @Override
+    public void onComplete(@NonNull Task<String> task) {
+        if (task.isSuccessful()) {
+            String token = task.getResult();
+            MobileCore.setPushIdentifier(token);
+        }
+    }
+});
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
-### Swift
+{% tab title="iOS (AEP 3.x)" %}
+
+To retrieve the push token in iOS, read the tutorial on [registering your application](https://developer.apple.com/documentation/usernotifications/registering_your_app_with_apns) within Apple's documentation. After retrieving the push token, use the following API to sync it with the Profile in AEP.
```

Comment:
Same comment here - keep the Platform, since AEP is against branding guidelines.